### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by nabramow**
Updating the contributing doc with the new info.

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
 # Developer Contributor Guide
 
-We collaborate on code through git, hosted on GitHub. If you are a software engineer (web/mobile/backend), you should request write access to the codebase.
+We collaborate on code through git, hosted on GitHub. If you are a software engineer (web/mobile/backend), you should request write access to the codebase. If you'd like to contribute regularly, reach out to @aapeliv or @bakeiro to join our Slack channel. This is where our main communication happens. We also do video meetings every week where we discuss priorities, architecture, etc.
```
Comment (nabramow) [inline docs/contributing.md]: I'm not sure if it's just Aapeli and David who can add someone to Slack so please correct me!

File: docs/contributing.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 
-- [Good first issues](https://github.com/Couchers-org/couchers/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
-- [Bugs in the backend: backend, bug](https://github.com/Couchers-org/couchers/issues?q=is%3Aopen+is%3Aissue+label%3Abackend+label%3Abug+)
-- [New features, design changes, bugs in the web frontend](https://github.com/Couchers-org/couchers/issues?q=is%3Aissue+label%3Aweb+is%3Aopen+sort%3Aupdated-desc)
 
+Tickets that are ready are under the "Ready" column. The higher the ticket, the bigger the priority, so try to choose from the top if you can. You can also choose a ticket from the "Abandoned" column if you want. These are tickets that were started, but abandoned and need someone to take them over the finish line.
+
+Once you begin a ticket, please assign yourself to it and move it into the "In Progress" column so we know who is working on what. 
+
+If you need clarification on a ticket, you can leave a comment and tag a contributor, or even better, leave a comment on our Slack Channel (reach out to @aapeliv or @bakeiro to join).
```
Comment (nabramow) [inline docs/contributing.md]: Same here let me know who should be tagged.

File: docs/contributing.md (hunk 1 above)
Comment (bakeiro) [reply docs/contributing.md]: I can't 😅

Comment (bakeiro) [reply docs/contributing.md]: but @aapeliv U can add me if U want ;)

**Comment by bakeiro** [comment]
so good U did this, this are the kind of small details that make the difference

File: docs/contributing.md (hunk 1 above)
Comment (nabramow) [reply docs/contributing.md]: Ahah whooops for some reason I thought you added me 😂.

**Comment by bakeiro** [comment]
build failed bc a timeout
image

performing empty commit :)

**Review by bakeiro — APPROVED** [review]

File: docs/contributing.md (hunk 1 above)
Comment (bakeiro) [reply docs/contributing.md]: yeah I sent the invitation to add you, it's just that another person has the final word, but usually it's pretty straighfordward

**Pull request merged**